taskchampion/xtask/src/main.rs

```diff
@@ -29,3 +45,99 @@ fn codegen() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+/// `cargo xtask msrv (X.Y)`
+///
+/// This updates all of the places in the repo where the MSRV occurs to <arg2> for the list of file paths defined as const in Xtask main.rs.
+/// If no pre-existing Xtask-formatted MSRV is found, inserts the MSRV as first comment block after the last `//!` library-doc comments.
+fn msrv(args: Vec<String>) -> anyhow::Result<()> {
+    match args.len(){
+        0 ..= 2 => println!("Error: xtask: Command `caro run xtask msrv` expects at least one argument, none detected."),
+        3 => {
+
+                // check that (X.Y) argument is (mostly) valid:
+                for xy in & mut args[2].chars() {
```

Comment:
This is fine, but somewhat more idiomatic to use an iterator expression: `if !args[2].chars().all(|c| c.is_numeric() || c == '.') { .. }`

Reply:
Done.